test/jasmine/tests/pie_test.js

```diff
@@ -731,6 +731,63 @@ describe('pie hovering', function() {
             })
             .then(done);
         });
+
+        it('should use hovertemplate if specified', function(done) {
+            Plotly.plot(gd, mockCopy.data, mockCopy.layout)
+            .then(_hover)
+            .then(function() {
+                assertLabel(
+                    ['4', '5', '33.3%'].join('\n'),
+                    ['rgb(31, 119, 180)', 'rgb(255, 255, 255)', 13, 'Arial', 'rgb(255, 255, 255)'],
+                    'initial'
+                );
+
+                return Plotly.restyle(gd, 'hovertemplate', '%{value}<extra></extra>');
+            })
+            .then(_hover)
+            .then(function() {
+                assertLabel(
+                    ['5'].join('\n'),
+                    null,
+                    'hovertemplate %{value}'
+                );
+
+                return Plotly.restyle(gd, {
+                    'text': [['A', 'B', 'C', 'D', 'E']],
+                    'hovertemplate': '%{text}<extra></extra>'
+                });
+            })
+            .then(_hover)
+            .then(function() {
+                assertLabel(
+                    ['E'].join('\n'),
+                    null,
+                    'hovertemplate %{text}'
+                );
+
+                return Plotly.restyle(gd, 'hovertemplate', '%{percent}<extra></extra>');
+            })
+            .then(_hover)
+            .then(function() {
+                assertLabel(
+                    ['33.3%'].join('\n'),
+                    null,
+                    'hovertemplate %{percent}'
+                );
+
+                return Plotly.restyle(gd, 'hovertemplate', '%{label}<extra></extra>');
```

Comment:
Could we add one arrayOk `hovertemplate` test for pies?

Reply:
Absolutely! There it is 1e4bd33